### DISCUSSION TITLE
How to implement something similar to JSON Server routes.js ?

### DISCUSSION THREAD

**Discussion by GeorgeFlorian** [parcel-bundler/parcel#6844] [Q&A]
I would like to implement something similar to the custom routes of JSON Server but I can't find any information on that.
Maybe it's already implemented, but I don't know how it's called.

Basically, I want multiple routes to point to the same endpoint.
Something like this:
```
{
    "/api/settings/change": "/change",
    "/api/settings/request": "/settings",
    "/api/info": "/info"
}
```
`/api/settings/request` and `/settings` go to the same page/endpoint.

Is this possible with Parcel ? And if it is, how can it be done ?

Thank you.

**Comment by mischnic** [comment]
You can hook into the Parcel dev server as described here: https://v2.parceljs.org/features/development/#api-proxy. If that's what you mean
